docs/source/api.rst

```diff
@@ -25,6 +25,26 @@ Client
 .. autoclass:: Profile
    :members:
 
+API 2 Client
```

Comment:
@carlosmondra could you review this?

Reply:
Do you mean the entire api.rst?

Reply:
the new section for the API 2 client. I realized they are not being shown in readthedocs. I guess this is correct. I want you to double check.